src/rgw/rgw_admin.cc

```diff
@@ -4817,7 +4817,11 @@ int main(int argc, char **argv)
     string user_str = user_id.to_str();
     int ret = store->cls_user_get_header(user_str, &header);
     if (ret < 0) {
-      cerr << "ERROR: can't read user header: " << cpp_strerror(-ret) << std::endl;
+      if (ret == -2) { /* in case of ENOENT */
+        cerr << "Error: There is no bucket/object in the cluster. Please create buckets/object first." << std::endl;
```

Comment:
Sounds to me more like 'User does not exist error, or user has not been initialized'